include/robot_localization/ros_filter.h

```diff
@@ -596,6 +591,21 @@ template<class T> class RosFilter
     //!
     boost::mutex diagnosticMapsMutex_;
 
+    //! @brief TimeToPublish object that determines the publishing time to maintain a constant rate
+    //!
+    RosFilterUtilities::TimeToPublish timeToPublish_;
+
+    //! @brief Frequency diagnostics updater
+    //!
+    boost::shared_ptr<diagnostic_updater::HeaderlessTopicDiagnostic> freqDiag_;
+
+    //! Minimum desired frequency
+    //!
+    double minFrequency_;
+
+    //! Maximum desired frequency
```

Comment:
Not sure why the other doxygen comments have the extra line with `//!`. Here is missed